plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/matcher/WithLabelMatcher.java

```diff
@@ -75,8 +75,15 @@ public boolean matches(Object item) {
 		if ((item instanceof List) || (item instanceof Text) || (item instanceof Button)
 				|| (item instanceof Combo) || (item instanceof CCombo) || (item instanceof Spinner)) {
 			String widgetLabel = WidgetLookup.getInstance().getLabel((Widget)item);
-			if (widgetLabel != null && matcher.matches(widgetLabel)) {
-				return true;
+			if (widgetLabel != null) {
+				// Ignore asterisk and spaces
+				String widgetLabel2 = widgetLabel.trim();
+				if (widgetLabel2.endsWith("*")) {
+					widgetLabel2 = widgetLabel2.substring(0, widgetLabel2.length() - 1).trim();
```

Comment:
My point was to use labels with or without the asterisk. 

There are tests which don't care which fields are required (here we can use labels without asterisk) but there are also tests which take care of which fields are required and which are not (here we need to use the asterisk).